src/chart/onLayout.js

```diff
@@ -43,39 +43,10 @@ export default function onLayout() {
             //Reset listing.
             context.listing.wrap.selectAll('*').remove();
             context.wrap.select('#listing-instruction').style('display', 'block');
-
-            //Sync status filter with legend items.
-            if (d.label === 'Status') {
-                const statusFilter = d3.select(this),
-                    selectedOptions = statusFilter.selectAll('.changer option:checked').data(), // selected statuses
-                    legendItems = context.wrap.selectAll('.legend-item').classed('selected', false), // de-select all legend items
-                    selectedLegendItems = legendItems
-                        .filter(d => selectedOptions.indexOf(d.label) > -1)
-                        .classed('selected', true); // sync legend items with status options
-
-                legendItems.each(function() {
-                    const legendItem = d3.select(this),
-                        selected = legendItem.classed('selected');
-                    legendItem.style({ background: selected ? 'lightgray' : 'white' });
-                });
-            }
+            context.listing.init(context.filtered_data);
         });
 
-    //Add download link.
-    if (this.config.exportData)
-        this.controls.wrap
-            .insert('a', ':first-child')
-            .attr('id', 'downloadCSV')
-            .style({
-                'text-decoration': 'underline',
-                color: 'blue',
-                cursor: 'pointer',
-                float: 'right',
-                margin: '5px',
-                padding: '5px',
-                clear: 'right'
-            })
-            .text('Download Query Data');
+    //where did download link go...
```

Comment:
wondering the same thing but since it's built into webcharts we don't need it anymore